src/conn/conn_cache_pool.c

```diff
@@ -674,8 +674,9 @@ __cache_pool_adjust(WT_SESSION_IMPL *session,
 			 * potentially a negative feedback loop in the
 			 * balance algorithm.
 			 */
-			smallest = (100 * __wt_cache_bytes_inuse(cache)) /
-			    cache->eviction_trigger;
+			smallest = (uint64_t)
```

Comment:
is this code structuring okay, please confirm with Alex

Reply:
The cast should be on the same line as the thing it is casting. If needed, break the line after the `=` instead.